pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -946,6 +970,15 @@ func (pl *dynamicResources) Reserve(ctx context.Context, cs *framework.CycleStat
 	return statusUnschedulable(logger, "waiting for resource driver to provide information", "pod", klog.KObj(pod))
 }
 
+func containsNode(hay []string, needle string) bool {
```

Comment:
https://pkg.go.dev/slices#Contains

Reply:
That is not available yet because master still uses 1.20, while "slices" was added in 1.21.

Reply:
ah, I see.

Reply:
/lgtm